index.md

```diff
@@ -161,7 +148,7 @@ Within the **ts_open_bitstream_unit()** payload, any four-byte sequence that sta
 ### 4.3 The AV1 Access Unit 
 
 An AV1 Access Unit consists of all OBUs, including headers, between the end of the last OBU associated
-with the previous frame, and the end of the last OBU associated with the current frame. An illustration is provided in the figure below for an IBPP group of pictures.
+with the previous frame, and the end of the last OBU associated with the current frame. An illustration is provided in the figure below for a group of pictures with frames predicted as follows :
```

Comment:
The term DFG is no longer defined but it is used in the figure. A short description of the figure could be added that mentions DFG as defined in AV1.

Reply:
Similarly the acronym TU in the image is not explained.